### DISCUSSION TITLE
Add swiper card to index.php file

### DISCUSSION THREAD

**Discussion by ersingo** [bootscore/bootscore#622] [Q&A] [answered]
Hi;

How can I add swiper permanently to index.php file.

**Comment by crftwrk** [comment, marked as answer]
Simply `echo` the shortcode in `index.php`:

```php
<?php echo do_shortcode('[bs-swiper-card type="post" category="cars, boats" order="ASC" orderby="date" posts="12"]'); ?>
```

Change shortcode to your values. Please mark this answer as solved.

**Comment by ersingo** [comment]
:man_with_turban: How did I not think of that? I'm sorry.
